app/src/components/project/TypeScriptProjectGuide.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+
+import { Heading, Text, View } from "@arizeai/components";
+
+import { ExternalLink } from "@phoenix/components";
+import { CodeWrap } from "@phoenix/components/code/CodeWrap";
+import { PythonBlockWithCopy } from "@phoenix/components/code/PythonBlockWithCopy";
+
+import { TypeScriptBlockWithCopy } from "../code/TypeScriptBlockWithCopy";
+
+import {
+  TypeScriptIntegrations,
+  TypeScriptPlatformIntegrations,
+} from "./Integrations";
+
+type PythonProjectGuideProps = {
+  /**
+   * An existing project name
+   */
+  projectName?: string;
+};
+
+const PHOENIX_OTEL_DOC_LINK =
```

Comment:
do we have a corresponding ts docs link like the ones above for python?

Reply:
It's super bare-bones right now - gotta work with you on the TS docs side of things